src/test/java/seedu/address/logic/commands/ListPersonCommandTest.java

```diff
@@ -15,7 +15,7 @@
 /**
  * Contains integration tests (interaction with the Model) and unit tests for ListCommand.
```

Comment:
```suggestion
 * Contains integration tests (interaction with the Model) and unit tests for ListPersonCommand.
```
Perhaps you can change this comment.